x/evm/state/balance_test.go

```diff
@@ -0,0 +1,90 @@
+package state
+
+import (
+	"math/big"
+	"testing"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/sei-protocol/sei-chain/x/evm/keeper"
+	"github.com/sei-protocol/sei-chain/x/evm/types"
+	"github.com/stretchr/testify/require"
+)
+
+func TestAddBalance(t *testing.T) {
```

Comment:
can we also add tests for 
- very large values (overflow)
- insufficient balances

Reply:
added